### ISSUE TITLE
Guessit issues with parsing releases from animetorrents

### ISSUE THREAD

**Issue by p0psicles** [pymedusa/Medusa#1747]
### Before submitting your issue:

Enable debug logging in Medusa settings, reproduce the error (be sure to disable after the bug is fixed)

**Branch/Commit: 7083ed5028ecfa0388d5229ab75fa5a9d86831d5**
**OS: windows 7**
**What you did: Searched for show Naruto**
**What happened: Provider is giving back results, but the releases are not parsed correctly.**
**What you expected: Parsed titles.**
**Logs:**
```
2016-12-07 22:48:11 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] GET URL: http://animetorrents.me/ajax/torrents_data.php?search=Naruto+Shippuuden%25476&searchin=filedisc&cat=2&SearchSubmit=&total=1&page=1 [Status: 200]
2016-12-07 22:48:11 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] Data returned from provider does not contain any torrents
2016-12-07 22:48:11 INFO     SEARCHQUEUE-DAILY-SEARCH :: [ad18671] Using daily search providers
2016-12-07 22:48:11 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [AnimeTorrents] :: [ad18671] Last update was too soon, using old cache: 2016-12-07 22:40:18. Updated less then 10 minutes ago
2016-12-07 22:48:11 DEBUG    SEARCHQUEUE-DAILY-SEARCH :: [AnimeTorrents] :: [ad18671] Transaction with 5 queries executed
2016-12-07 22:48:11 INFO     SEARCHQUEUE-DAILY-SEARCH :: [ad18671] No needed episodes found
2016-12-07 22:48:11 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] GET URL: http://animetorrents.me/ajax/torrents_data.php?search=Naruto+Shippuuden%25476&searchin=filedisc&cat=7&SearchSubmit=&total=1&page=1 [Status: 200]
2016-12-07 22:48:11 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] Found result: Naruto Shippuuden - 476-479 (2007) [HorribleSubs][WEBRip]  [HD 720p] with 12 seeders and 2 leechers
2016-12-07 22:48:11 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] Search string: NARUTO 疾風伝%476
2016-12-07 22:48:12 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] GET URL: http://animetorrents.me/ajax/torrents_data.php?search=NARUTO+%E7%96%BE%E9%A2%A8%E4%BC%9D%25476&searchin=filedisc&cat=2&SearchSubmit=&total=1&page=1 [Status: 200]
2016-12-07 22:48:12 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] Data returned from provider does not contain any torrents
2016-12-07 22:48:12 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] GET URL: http://animetorrents.me/ajax/torrents_data.php?search=NARUTO+%E7%96%BE%E9%A2%A8%E4%BC%9D%25476&searchin=filedisc&cat=7&SearchSubmit=&total=1&page=1 [Status: 200]
2016-12-07 22:48:12 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] Data returned from provider does not contain any torrents
2016-12-07 22:48:12 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] Search string: Naruto Shippuden%476
2016-12-07 22:48:12 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] GET URL: http://animetorrents.me/ajax/torrents_data.php?search=Naruto+Shippuden%25476&searchin=filedisc&cat=2&SearchSubmit=&total=1&page=1 [Status: 200]
2016-12-07 22:48:12 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] Data returned from provider does not contain any torrents
2016-12-07 22:48:13 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] GET URL: http://animetorrents.me/ajax/torrents_data.php?search=Naruto+Shippuden%25476&searchin=filedisc&cat=7&SearchSubmit=&total=1&page=1 [Status: 200]
2016-12-07 22:48:13 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] Data returned from provider does not contain any torrents
2016-12-07 22:58:07 WARNING  DAILYSEARCHER :: [ad18671] Manual search is running. Can't start Daily search
2016-12-07 22:59:43 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [AnimeTorrents] :: [ad18671] Unable to match Naruto.Shippuuden.-.476-479.(2007).[HorribleSubs][WEBRip]..[HD.720p] to a show in your database. Parser result: title: Naruto Shippuuden, alias: Naruto Shippuuden - 476-479, alternative_title: 476-479, year: 2007, release_group: HorribleSubs, format: WEBRip, other: HD, screen_size: 720p, type: episode, parsing_time: 0.214999914169, total_time: 682.592000008, absolute_episode: [], quality: 720p WEB-DL, episode: [], season: None
2016-12-07 22:59:43 DEBUG    SEARCHQUEUE-MANUAL-79824 :: [ad18671] 79824: Formatting pattern: %SN - S%0SE%0E - %EN -> Naruto Shippuden - S22E18 - The Final Battle
2016-12-07 22:59:43 INFO     SEARCHQUEUE-MANUAL-79824 :: [ad18671] Unable to find manual results for: [Naruto Shippuden - S22E18 - The Final Battle]
```
'Naruto.Shippuuden.-.476-479.(2007).[HorribleSubs][WEBRip]..[HD.720p]'

Report guessit:
```
D:\medusa\tests>python report_guessit.py 'Naruto.Shippuuden.-.476-479.(2007).[HorribleSubs][WEBRip]..[HD.720p]'
# guessit: 2.1.1.dev0  rebulk: 0.8.3.dev0
? 'Naruto.Shippuuden.-.476-479.(2007).[HorribleSubs][WEBRip]..[HD.720p]'
: title: 'Naruto Shippuuden
  alias: 'Naruto Shippuuden - 476-479
  alternative_title: 476-479
  year: 2007
  format: WEBRip
  other: HD
  screen_size: 720p
  type: episode
  parsing_time: 0.0829999446869
```

What i expect:
```
D:\medusa\tests>python report_guessit.py 'Naruto.Shippuuden.-.476-479.(2007).[HorribleSubs][WEBRip]..[HD.720p]'
# guessit: 2.1.1.dev0  rebulk: 0.8.3.dev0
? 'Naruto.Shippuuden.-.476-479.(2007).[HorribleSubs][WEBRip]..[HD.720p]'
: title: 'Naruto Shippuuden
  alias: ''
  alternative_title: ''
  year: 2007
  format: WEBRip
  other: HD
  screen_size: 720p
  type: episode
  absolute_season: 1
  absolute_episode: [476, 476, 478, 479]
  parsing_time: 0.0829999446869
```

Not totally sure if guessit should report it like that.
Those eps are actually absolute episodes, but don't know if guessit can guess that.

**Comment by p0psicles** [comment]
A log with some more parse results.
Allot of them are not parsed correct:
https://gist.github.com/p0psicles/f5350eba0b78c2b8c085ab6a24b07346

**Comment by p0psicles** [comment]
```
? 'Joker.Game.-.07.(2016).[RH].[English.Dubbed][WEBRip]..[HD.1080p]'
: title: 'Joker Game
  alternative_title: 07
  year: 2016
  language: en
  format: WEBRip
  other: HD
  screen_size: 1080p
  type: episode
  parsing_time: 0.0600001811981
```
Should be:
: title: 'Joker Game'
  alternative_title: ''
  year: 2016
  language: en
  format: WEBRip
  other: HD
  screen_size: 1080p
  type: episode
  season: 01
  episode: 07
  release_group: RH
  parsing_time: 0.0600001811981
```

**Comment by p0psicles** [comment]
```? 'Hibike!.Euphonium.2.-.10.(2016).[HorribleSubs][WEBRip]..[HD.720p]'
: title: 'Hibike! Euphonium 2
  alias: 'Hibike! Euphonium 2 - 10
  alternative_title: 10
  year: 2016
  format: WEBRip
  other: HD
  screen_size: 720p
  type: episode
  parsing_time: 0.0599999427795```

Should be: 
```
? 'Hibike!.Euphonium.2.-.10.(2016).[HorribleSubs][WEBRip]..[HD.720p]'
: title: 'Hibike! Euphonium 2
  alias: ''
  alternative_title: ''
  year: 2016
  format: WEBRip
  other: HD
  screen_size: 720p
  type: episode
  release_group: HorribleSubs
  absolute_season: 01
  absolute_episode: 10
  parsing_time: 0.0599999427795
```

**Comment by ratoaq2** [comment]
It should be fixed by #1755
